modules/service/src/main/resources/db/migration/V0854__program_reference_update.sql

```diff
@@ -0,0 +1,3 @@
+-- Add the new types to the program type enum
+ALTER TYPE e_program_type ADD VALUE 'commissioning' BEFORE 'engineering';
+ALTER TYPE e_program_type ADD VALUE 'monitoring'    BEFORE 'science';
```

Comment:
I've learned my lesson and put these in a separate migration (i.e., separate transaction).